msbuild/Makefile

```diff
@@ -373,6 +379,7 @@ all-local:: $(MSBUILD_PRODUCTS) .stamp-test-xml
 
 # make all the target assemblies build when any of the sources have changed
 $(foreach dll,$(MSBUILD_TASK_ASSEMBLIES),$(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).dll $(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).pdb): .build-stamp
+	$(foreach dll,$(TRANSLATED_ASSEMBLIES),Xamarin.Localization.MSBuild/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).resources.dll): .build-stamp
```

Comment:
The indentation is wrong here (whitespace is significant in makefiles): there should be no indentation for this line.

Reply:
Okay should be fixed now!